examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -59,102 +79,269 @@ class TravelDestinationItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final ThemeData theme = Theme.of(context);
-    final TextStyle titleStyle = theme.textTheme.headline.copyWith(color: Colors.white);
-    final TextStyle descriptionStyle = theme.textTheme.subhead;
+    return SafeArea(
+      top: false,
+      bottom: false,
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Column(
+          children: <Widget>[
+            const SectionTitle(title: 'Card with actions'),
+            Container(
+              height: height,
+              child: Card(
+                clipBehavior: Clip.antiAlias,
+                shape: shape,
+                child: TravelDestinationContent(destination: destination),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
 
+class TappableTravelDestinationItem extends StatelessWidget {
+  TappableTravelDestinationItem({ Key key, @required this.destination, this.shape })
+      : assert(destination != null && destination.isValid),
+        super(key: key);
+
+  static const double height = 312.0;
```

Comment:
Since all the cards are in a list, it doesn't have to all fit on screen since it will be scrollable anyways. I added a comment clarifying what this is for.